tests/elm-package.json

```diff
@@ -1,20 +1,20 @@
 {
     "version": "1.0.0",
-    "summary": "Sample Elm Test",
-    "repository": "https://github.com/user/project.git",
-    "license": "BSD-3-Clause",
+    "summary": "Test Suites",
+    "repository": "https://github.com/elm-community/elm-time.git",
+    "license": "BSD3",
     "source-directories": [
-        ".",
-        "../src"
+        "../src",
+        "."
     ],
     "exposed-modules": [],
     "dependencies": {
+        "elm-lang/core": "5.0.0 <= v < 6.0.0",
+        "elm-community/elm-test": "4.0.0 <= v < 5.0.0",
         "Bogdanp/elm-combine": "3.0.0 <= v < 4.0.0",
-        "elm-community/elm-test": "3.0.0 <= v < 4.0.0",
         "elm-community/json-extra": "2.0.0 <= v < 3.0.0",
-        "elm-lang/core": "5.0.0 <= v < 6.0.0",
         "elm-lang/lazy": "2.0.0 <= v < 3.0.0",
-        "rtfeldman/node-test-runner": "3.0.0 <= v < 4.0.0"
+        "elm-lang/lazy": "2.0.0 <= v < 3.0.0"
```

Comment:
Hi Trevor, thanks for following up from our meetup last night.

Could you delete the duplicate line for `elm-lang/lazy` and I'll do the merge.  I had this ticket to do this morning, and so thanks for doing it for me.